main.go

```diff
@@ -331,7 +332,14 @@ func sanityCheck(args *arguments) exitCode {
 		var minMajor, minMinor uint32
 		switch runtime.GOARCH {
 		case "amd64":
-			minMajor, minMinor = 4, 19
+			if args.verboseMode {
+				log.Infof("The debug version of the tracer significantly increases the number " +
+					"of BPF instructions, and at least kernel version 5.2 is required to " +
+					"ensure that BPF verifier is happy")
```

Comment:
I would remove this as it may confuse (it's also printed in release builds). We log a failure error later on.

```suggestion
```